src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_hana_commands.py

```diff
@@ -65,7 +65,7 @@ def test_backup_wl_hana_archive (self):
 
         is_move = False
         for i in rp_names:
-            if i['tierType']=="VaultStandard" and i['properties']['recoveryPointMoveReadinessInfo']['ArchivedRP']['isReadyForMove']==True:
+            if i['tierType'] == "VaultStandard" and i['properties']['recoveryPointMoveReadinessInfo']['ArchivedRP']['isReadyForMove']:
```

Comment:
`is True` should be kept, according to https://github.com/Azure/azure-cli/pull/30406/files#r1901091313:

```suggestion
            if i['tierType'] == "VaultStandard" and i['properties']['recoveryPointMoveReadinessInfo']['ArchivedRP']['isReadyForMove'] is True:
```

Reply:
I have reverted these two lines to `is True`. So I hope it's ready to be merged soon :)

Reply:
@kairu-ms You'll need to :+1:  again